vspace/src/main/java/edu/asu/diging/vspace/web/publicview/IPreviewConstant.java

```diff
@@ -0,0 +1,5 @@
+package edu.asu.diging.vspace.web.publicview;
+
+public interface IPreviewConstant {
```

Comment:
just call this `ExhibitionConstants` (note no `I` and plural of constant), since we might need more constants in the future.